sites/game-client/map-renderer.mjs

```diff
@@ -186,7 +224,11 @@ class MapRenderer {
     for (const [, layerName, renderFunction, renderArgs] of this.customizedLayers) {
       if (renderFunction === '_drawCharacters') {
         this._drawCharacters(renderArgs);
-      } else {
+      } 
+      else if (renderFunction === '_drawWatermark') {
+        this._drawWatermark(renderArgs);
+      }
+      else {
```

Comment:
I remember that our eslint prefers
```
} else {
```
over
```
}
else {
```

Correct me if I'm wrong :pray:

Reply:
IDK, but I fixed it